.vale/styles/AsciiDoc/MatchingDotCallouts.yml

```diff
@@ -0,0 +1,41 @@
+---
+extends: script
+ignorecase: true
+message: "Corresponding callout not found."
+level: error
+link: https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/
+scope: raw
+script: |
+  text := import("text")
+  matches := []
+
+  num_codeblock_callouts := 0
+  num_callouts := 0
+  codeblock_callout_regex := ".+<(\\.)>"
+  callout_regex := "^<(\\.)>"
+
+  for line in text.split(scope, "\n") {
+    if text.re_match(codeblock_callout_regex, line) {
+      //restart for new listingblock
+      num_callouts = 0
+      //account for lines with multiple callouts
+      num_callouts_in_line := text.count(line, "<.>")
+      if num_callouts_in_line > 1 {
+        num_codeblock_callouts = num_codeblock_callouts + num_callouts_in_line
+      } else{
+        num_codeblock_callouts++
+      }
+    }
+
+    if text.re_match(callout_regex, line) {
+      num_callouts++
+      if num_callouts > num_codeblock_callouts {
+        start := text.index(scope, line)
+        matches = append(matches, {begin: start, end: start + len(line)})
+      }
```

Comment:
I guess there's no easy way to do the reverse, check if codeblock callouts are greater than callouts... perhaps one to think about in the future.

Reply:
Could be done, but I just followed how the AsciiDoctor validation works